### ISSUE TITLE
How to use it 

### ISSUE THREAD

**Issue by tzw-1998** [unsky/focal-loss#2]
please give a demo

**Comment by unsky** [comment]
i have update README

**Comment by Breakmydream** [comment]
@unsky 
As your said
i use it in cifar10 dataset：
`cost = mx.sym.Custom(op_type='FocalLoss',name='cost',data=logits,label=label,gamma=2)`
but 
![l8u9g_f9b pp3_qv3 f f e](https://user-images.githubusercontent.com/29767139/29318718-db01fabe-8203-11e7-8c13-9e5b972dc0bc.png)

**Comment by tzw-1998** [comment]
@unsky 
As your said
 i use it in cifar10 dataset：
`cost = mx.sym.Custom(op_type='FocalLoss',name='cost', data=logits, labels=label, gamma= 2)`
but
![j o3chpbk o xk 5b 9 tv](https://user-images.githubusercontent.com/29767139/29318993-e331e5e0-8204-11e7-9897-544db2d877b4.png)

**Comment by tzw-1998** [comment]
someti is this error
![qu2pf jekw 9my7 52k yj](https://user-images.githubusercontent.com/29767139/29319275-093035f2-8206-11e7-9ae6-37b552dff3e0.png)

**Comment by unsky** [comment]
i think this error is caused by mxnet version . in my experiment  mxnet4.9 and python2.7 are used

**Comment by unsky** [comment]
@Breakmydream  the code is very simple， you can fix it in your env.

**Comment by tzw-1998** [comment]
I only find the mxnet version 0.10.0 or 0.11.0rc in the mxnet.io/get_started/install.html

**Comment by unsky** [comment]
@Breakmydream  sorry https://github.com/apache/incubator-mxnet/tree/62ecb60 v0.9.x

**Comment by smmdream** [comment]
I have met a question and spent a afternoon trying to solve it,but I failed.could you tell me what is "parser.add_argument('--frequent', help='frequency of logging', default=config.default.frequent, type=int)" need,I run the code,and met error "usage: train.py [-h] --cfg CFG
train.py: error: argument --cfg is required",could you help me.

**Comment by unsky** [comment]
。。。。。。
python train.py  --cfg pascal_voc.yml
